content/_index.md

```diff
@@ -2,8 +2,8 @@
 
 ## About the BAPC
 
-The Benelux Algorithm Programming Contest (BAPC) is a contest in which about 100 teams from leading universities in Luxembourg, Belgium and the Netherlands are served a series of algorithmic problems/puzzles. The goal of each team is to solve the most puzzles as possible within the set time limit. Solutions need to be programmed out on a computer and can be submitted to a semi-automated judging system, after which the solution is checked.
+The Benelux Algorithm Programming Contest (BAPC) is a contest in which teams from leading universities in Luxembourg, Belgium and the Netherlands are served a series of algorithmic problems/puzzles. The goal of each team is to solve the most puzzles as possible within the set time limit. Solutions need to be programmed out on a computer and can be submitted to a semi-automated judging system, after which the solution is checked.
 
 The teams that have solved the most puzzles at the end of the contest qualify for the [Northwestern European Regional Contest (NWERC)](https://www.nwerc.eu). In this regional competition the teams can qualify themselves for a ticket to the [International Collegiate Programming Contest (ICPC)](https://icpc.global), also known as the World Finals.
 
-The [Vrije Universiteit Amsterdam](https://www.vu.nl/) hosts the BAPC 2021. A committee of [study association STORM](https://storm.vu) organized the event.
+The [Vrije Universiteit Amsterdam](https://www.vu.nl/) hosts the BAPC 2021, organised by a committee of [study association STORM](https://storm.vu).
```

Comment:
The spell checker doesn't like British spelling

Reply:
Yes, see: https://github.com/sylvainhalle/textidote#spelling-grammar-and-style

and change: https://github.com/StudieverenigingSTORM/BAPC21-site/blob/edits_alex/.github/workflows/grammar.yml#L25-L40

I'm fine with either en-gb or en-us, but we should pick one OR we can make it complain only when both disagree.

As you and @iMylene will probably update the most I think the choice is up to you two.